blockdag/validate.go

```diff
@@ -601,8 +608,12 @@ func (dag *BlockDAG) checkBlockHashMerkleRoot(block *util.Block) error {
 	return nil
 }
 
-func (dag *BlockDAG) checkBlockDuplicateTransactions(transactions []*util.Tx) error {
+func (dag *BlockDAG) checkBlockDuplicateTransactions(block *util.Block) error {
+	// Check for duplicate transactions. This check will be fairly quick
+	// since the transaction IDs are already cached due to building the
+	// merkle tree above.
```

Comment:
What is above? it's two separate funcitons? Maybe move this comment to outer scope

Reply:
Bottom line: I don't mind this either way, either revise to something Ori agrees to, or remove this comment